org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4graph/GraphViewer.java

```diff
@@ -61,6 +61,8 @@
  * This class displays the graph.
  * <p>
  * <b>Thread Safety:</b> Can be called only by the AWT event thread.
+ *
+ * @modified: Nuno Macedo // [HASLab] electrum-vizualizer
```

Comment:
since the vizualizer is now multi-pane, this is needed to preserve the scale of the graphs when navigating the instance